src/combination.jl

```diff
@@ -252,3 +250,32 @@ function plot_order_temporalCombin(params1, params2, g1s, g2s, named1, named2)
         title = string(named1, " --> ", named2, " - ", named2, " --> ", named1),
     )
 end
+
+####--------------- Loewe additivity ------------------####
+""" Find the inverse of a hill function (concentration), given the parameters and the effect. """
+function inv_hill(p::Array{Float64, 1}, y)
+    #p = [EC50, min, max, steepness], y:effect. it returns concentration
+    conc = p[1] * (((y - p[2]) / (p[3] - y)) ^ p[4])
+    return conc
+end
+
+""" The combination Index for Loewe. """
+function loewe(d1, p1, d2, p2, conc1, conc2)
+    # x: the combined effect
+    f(x) = (d1 / inv_hill(p1, x)) + (d2 / inv_hill(p2, x)) - 1.0
+
+    combined_effect = find_zero(f, [(minimum([p1[2], p2[2]]) + eps()), (maximum([p1[3], p2[3]])-eps())])
+    return combined_effect
+end
```

Comment:
Here I wrote the combination index for the Loewe. 
The lower and upper bound to find the roots are set to be between the min and max parameters, due to the kth root in the inverse Hill function. and the epsilons are just to avoid being equal to min and max since it creates zeros in the denominator.